docs/conf.py

```diff
@@ -93,6 +93,24 @@
 # List of notebooks NOT to execute (use output stored in notebook instead)
 execution_excludepatterns = []
 
+
+# sphinx-notfound-page
+# https://github.com/readthedocs/sphinx-notfound-page
+# Renders a 404 page with absolute links
+import importlib
+
+if importlib.util.find_spec("notfound"):
+    extensions.append("notfound.extension")
+
+    notfound_context = {
+        "title": "404: File Not Found",
+        "body": """
+    <h1>404: File Not Found</h1>
+    <p>Sorry, we couldn't find that page.</p>
```

Comment:
(not blocking) Could we add something like "This usually happens as a result of following an outdated link. Please ensure you're looking at the [most recent](https://open-forcefield-toolkit.readthedocs.io/en/) documentation unless you are deliberately trying to use an older version."